meta-lmp-base/recipes-security/optee/optee-examples-fio.inc

```diff
@@ -5,9 +5,7 @@ HOMEPAGE = "https://github.com/linaro-swg/optee_examples"
 LICENSE = "BSD-2-Clause"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=cd95ab417e23b94f381dafc453d70c30"
 
-DEPENDS = "optee-client virtual/optee-os python3-pycryptodomex-native"
-
-inherit python3native
+DEPENDS = "optee-client optee-os-tadevkit"
```

Comment:
Only for my context: the virtual/optee-os its been replaced with optee-os-tadevkit ?

Reply:
Yes, before we were using optee-os for providing the TA devkit, which is now separated into another recipe to allow  building and including external TAs as early TAs (integrated into the optee-os binary that gets loaded on early boot).

We don't need the virtual anymore because tadevkit will include the right recipe based on the provider.

Reply:
thanks for explaining